components/badge/badge.html

```diff
@@ -53,26 +53,26 @@
                         <li><a href="/components/list/list.html">List</a></li>
                         <li><a href="/components/navigation/navigation.html">Navigation</a></li>
                         <li><a href="/components/modal/modal.html">Modal</a></li>
-                        <li><a href="#">Rating</a></li>
+                        <!-- <li><a href="#">Rating</a></li>
                         <li><a href="#">Tost</a></li>
                         <li><a href="#">Grid</a></li>
-                        <li><a href="#">Slider</a></li>
+                        <li><a href="#">Slider</a></li> -->
```

Comment:
You can remove these comments

Reply:
Hey, Ajit. I have to create those but not doing them right now because of deadline constraints. Just focusing bare minimum for now.

Ah, yes conflict is because I have to change few code lines I have already merged in my dev(main) branch. I'll resolve that during the merge.

Reply:
Please approve if all good.